frontend/server/src/DAO/ACLs.php

```diff
@@ -10,4 +10,44 @@
  * @access public
  */
 class ACLs extends \OmegaUp\DAO\Base\ACLs {
+        /**
+     * Get all ACLs owned by a user along with their type, alias, and an empty users array.
+     *
+     * @param int $userId The user ID whose ACLs are being fetched.
+     * @return list<array{acl_id: int, type: string, alias: string, users: list<array<string, never>>}>
```

Comment:
Ok, está perfecto que se declare desde inicio `users`. Lo que faltaría es agregar el tipo correcto, si ya sabes que es lo que contendrá este elemento:
```suggestion
     * @return list<array{acl_id: int, type: string, alias: string, users: list<array{user_id: int, username: string, role_id: int, role_name: string, role_description: string}>}>
```

Reply:
updated in lastest commit